### ISSUE TITLE
Update Bahasa Malaysia translations

### ISSUE THREAD

**Issue by belajarpowershell** [squidfunk/mkdocs-material#6777]
### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "ms",
  "action.edit": "Edit halaman ini",
  "action.skip": "Langkau tajuk talian",
  "action.view": "Paparkan sumber laman ini",
  "announce.dismiss": "Jangan paparkan lagi",
  "blog.archive": "Arkib",
  "blog.categories": "Kategori",
  "blog.categories.in": "masuk",
  "blog.continue": "Continue reading ⬅️",
  "blog.draft": "Draft ⬅️",
  "blog.index": "Back to index ⬅️",
  "blog.meta": "Metadata ⬅️",
  "blog.references": "Related links ⬅️",
  "clipboard.copy": "Salin ke papan keratan",
  "clipboard.copied": "Disalin ke papan keratan",
  "consent.accept": "Accept ⬅️",
  "consent.manage": "Manage settings ⬅️",
  "consent.reject": "Reject ⬅️",
  "footer": "Pengaki",
  "footer.next": "Next ⬅️",
  "footer.previous": "Sebelumnya",
  "header": "Pengepala",
  "meta.comments": "Komen",
  "meta.source": "Sumber",
  "nav": "Navigasi",
  "readtime.one": "1 min read ⬅️",
  "readtime.other": "# min read ⬅️",
  "rss.created": "RSS feed ⬅️",
  "rss.updated": "RSS feed of updated content ⬅️",
  "search": "Search ⬅️",
  "search.placeholder": "Cari",
  "search.share": "Share ⬅️",
  "search.reset": "Padam",
  "search.result.initializer": "Siap carian",
  "search.result.placeholder": "Taip untuk mula mencari",
  "search.result.none": "Tiada dokumen yang sepadan",
  "search.result.one": "1 dokumen yang sepadan",
  "search.result.other": "# dokumen yang sepadan",
  "search.result.more.one": "1 lagi di halaman ini",
  "search.result.more.other": "# lagi di halaman ini",
  "search.result.term.missing": "Hilang",
  "select.language": "Pilih bahasa",
  "select.version": "Pilih versi",
  "source": "tajuk talian asal",
  "source.file.contributors": "Contributors ⬅️",
  "source.file.date.created": "tarikh fil asal dicipta",
  "source.file.date.updated": "Tarikh fil dikemas kini",
  "tabs": "Tab",
  "toc": "Jadual kandungan",
  "top": "Kembali ke atas"
}[key] }}{% endmacro %}
```

### Country flag

:flag_my:

### Before submitting

- [X] I've checked the list of [available languages](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language) for existing translations.
- [X] I assure that the translations are accurate to the best of my knowledge.
- [ ] __Optional__: I want to integrate this translation myself and create a pull request following the [contribution guide](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md).

**Comment by squidfunk** [comment]
Thanks! Added in 721eb6a. Would you like to add the remaining missing translations as well?

**Comment by belajarpowershell** [comment]
Sure let me submit for the remaining ones.

    On Monday, February 12, 2024 at 08:06:46 PM GMT+8, Martin Donath ***@***.***> wrote:  
 
 


Thanks! Added in 721eb6a. Would you like to add the remaining missing translations as well?

**Comment by squidfunk** [comment]
Released as part of [9.5.10](https://squidfunk.github.io/mkdocs-material/changelog/#9.5.10).

**Comment by squidfunk** [comment]
If you want to add the remaining translations, please create a new issue, as this issue is considered resolved.
